dev/com.ibm.ws.security.jwt_fat.builder/publish/servers/com.ibm.ws.security.jwt_fat.builder/configs/server_minimumConfig.xml

```diff
@@ -4,6 +4,7 @@
 	<featureManager>
 		<feature>jwt-1.0</feature>
 		<feature>jsonp-1.0</feature>
+		<feature>servlet-3.1</feature> 
```

Comment:
Why was this needed now? I'm assuming it must be related to switching out features between EE8, EE9, and EE10?

Reply:
yeah - we need it for EE9 and EE10 - forcing servlet 5 & 6 through the transform ends up adding it in places that we don't need it (and results in multiple copies of each)